src/components/shift-schedule/RegisterSection.tsx

```diff
@@ -0,0 +1,23 @@
+import { Button } from '@/components/ui/Button';
+import { Checkbox } from '@/components/ui/Checkbox';
+import { Label } from '@/components/ui/Label';
+import { cx } from '@/lib/utils';
+import { useTranslations } from 'next-intl';
+
+function RegisterSection({ className }: { className?: string }) {
+  const t = useTranslations(
+    'shiftSchedule.scheduleTable.scheduleCell.scheduleCellDialog.registerSection',
+  );
+
+  return (
+    <div className={cx(className, 'space-y-3')}>
+      <section className='flex gap-2'>
+        <Label htmlFor='recurring'>{t('recurring')}: </Label>
+        <Checkbox id='recurring' />
+      </section>
```

Comment:
```suggestion
      <div className='flex gap-2'>
        <Label htmlFor='recurring'>{t('recurring')}: </Label>
        <Checkbox id='recurring' />
      </div>
```
Or another element that you find fitting

Reply:
A name change for the component itself can be useful as well, for example `RegisterShiftAction` or just `RegisterShift` in case `Action` should be reversed for something else. @michaelbrusegard probably knows